tests/workspace_lint_config.rs

```diff
@@ -0,0 +1,26 @@
+use assert_cmd::Command;
+
+#[test]
+fn test_workspace_config() {
+    // TODO: we are only checking for required semver version overrides
+    // right now because that is all that is currently implemented.
+    let mut cmd = Command::cargo_bin("cargo-semver-checks")
+        .expect("cargo semver-checks command should exist");
+
+    cmd.current_dir("test_crates/workspace_overrides").args([
+        "semver-checks",
+        "--baseline-root",
+        "baseline",
+        "--manifest-path",
+        "current/pkg/Cargo.toml",
```

Comment:
Could you either make sure to either make a new subdirectory for all test cases for package/workspace config testing, or follow the existing norms in the directory where you're adding the test crates?

Using `baseline/current` instead of `old/new` will probably break some of the tooling we use, such as this script for example: https://github.com/obi1kenobi/cargo-semver-checks/blob/f8f89ecfa7f1e60b343e7b07beaf415393f244f8/scripts/regenerate_test_rustdocs.sh#L46-L48

Reply:
We can't use the `regenerate_test_rustdocs.sh` because we are passing the manifest path to `cargo-semver-checks`, so we don't want to generate rustdoc for these crates, which is why I didn't use new and old.  It does make more sense to move them into a subdirectory, though.